client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -255,19 +255,17 @@ public void start() throws MQClientException {
     }
 
     private void startScheduledTask() {
-        if (null == this.clientConfig.getNamesrvAddr()) {
```

Comment:
in fact, even though we specified the web endpoint, client will not fetch name server addresses any more beacase of below code in start():

`// If not specified,looking address from name server
                    if (null == this.clientConfig.getNamesrvAddr()) {
                        this.mQClientAPIImpl.fetchNameServerAddr();
                    }`